### ISSUE TITLE
International Committee on Taxonomy of Viruses

### ISSUE THREAD

**Issue by arw36** [ropensci/taxize#695]
There are [downloadedable species list](https://talk.ictvonline.org/files/master-species-lists/), [historical taxonomies](https://talk.ictvonline.org/taxonomy/), and [open reports](https://talk.ictvonline.org/ictv-reports/). No current API. I think @noamross has ideas on ways to make into package, but may be more appropriate to include here.

**Comment by noamross** [comment]
Yes, this is the project for which I was asking about [time-versioned taxonomies over in the taxa repo](https://github.com/ropensci/taxa/issues/136).  ICTV has no API and doesn't deliver the data in bulk, but I have a scraper that uses the website's internal API.  They have a generous `robots.txt` and all the non-data text is CC-BY-SA 4.0.   Probably the thing to do would be to set up a bulk data set of scraped data, maybe have a package for it but also point taxize to the raw data.  If so, what format should the raw bulk data be in, @sckott?

**Comment by sckott** [comment]
thanks @arw36 and @noamross 

For bulk data, tabular is of course easy to work with, but we can also parse json or xml here if needed.

**Comment by noamross** [comment]
This is definitely in the "need an intern/volunteer to make it happen" category.

**Comment by sariya** [comment]
I'd like to volunteer, any starting points? But, I've never contributed to opensource projects before. For sure, I'd have difficult time to understand work-flow of connections in the beginning.
